streamer/src/quic.rs

```diff
@@ -517,6 +519,16 @@ impl StreamStats {
                 self.connection_rate_limiter_length.load(Ordering::Relaxed),
                 i64
             ),
+            (
+                "outstanding_incoming_handshakes",
+                self.outstanding_incoming_handshakes.load(Ordering::Relaxed),
```

Comment:
Why not use swap 0 so that we track the counts within the sampling period?

Reply:
It's supposed to be the number of outstanding (in-process) handshakes at the time of sample, so not the number of new ones since the last sample. I think this is more representative and reliable, as we can't guarantee a consistent sampling period.